sky/data/storage.py

```diff
@@ -1183,16 +1220,24 @@ def _delete_s3_bucket(self, bucket_name: str) -> None:
         try:
             with log_utils.safe_rich_status(
                     f'[bold cyan]Deleting S3 bucket {bucket_name}[/]'):
-                subprocess.check_output(remove_command.split(' '))
+                subprocess.check_output(remove_command.split(' '),
+                                        stderr=subprocess.STDOUT)
         except subprocess.CalledProcessError as e:
-            logger.error(e.output)
-            with ux_utils.print_exception_no_traceback():
-                raise exceptions.StorageBucketDeleteError(
-                    f'Failed to delete S3 bucket {bucket_name}.')
+            if 'NoSuchBucket' in e.output.decode('utf-8'):
+                logger.debug(
```

Comment:
As we have L1026, can we remove this message?

Reply:
Ah, this was intentionally left in as `logger.debug` (which is not shown to user, unlike `logger.info`). It should be fine to have verbose logging in debug.